<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerArrowWidgetBase</class>
 <widget class="QWidget" name="QgsComposerArrowWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>214</width>
        <height>383</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Arrow</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="mArrowColorButton">
         <property name="text">
          <string>Arrow color...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="mOutlineWidthSpinBox">
         <property name="prefix">
          <string>Line width </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="mArrowHeadWidthSpinBox">
         <property name="prefix">
          <string>Arrow head width </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="mArrowMarkersGroupBox">
         <property name="title">
          <string>Arrow markers</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="mStartMarkerLabel">
            <property name="text">
             <string>Start marker</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="mStartMarkerLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="mStartMarkerToolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QRadioButton" name="mSvgMarkerRadioButton">
            <property name="text">
             <string>SVG markers</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="mEndMarkerLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="mEndMarkerToolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="mEndMarkerLabel">
            <property name="text">
             <string>End marker</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="mNoMarkerRadioButton">
            <property name="text">
             <string>No marker</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="mDefaultMarkerRadioButton">
            <property name="text">
             <string>Default marker</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>62</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
